### PR TITLE
Bump Red Hat distribution to version 1.9

### PR DESCRIPTION
**Pull request by rimolive**
We would like to propose bump Red Hat distribution to Kubeflow 1.9 as part of updating the Distributions table.

## Why we are requesting this change?

Red Hat has been actively contributing with Kubeflow since 1.9 release in multiple Working Groups, including the new component: Model Registry.

Currently, Red Hat OpenShift AI uses the following components from Kubeflow 1.9:

* Pipelines
* Training Operator (upstream project currently)
* Model Registry
* Notebooks (in progress)

Our commitment with the community has confirmed by our initial contributions for the upcoming Kubeflow 1.10 release, and we've been working on Notebooks 2.0 (expected to be released along with Kubeflow 1.10) as well.

Components we are not considering or still evaluating:

* Central Dashboard (we implemented our own dashboard)
* Profile Controller/Volumes Web app (we use our own implementation under dashboard)
* Tensorboard Controller/Tensorboard Web app (we don't have plans to support)
* Katib (we are still evaluating adoption)

Because we don't have a clear process to evaluate if a Distribution is ready to support a given KF release, we understand that these arguments could be considered to state our current contribution level within the community. We are also happy to contribute with a concrete Conformance Testing process in which other distributions can work on a better workflow to promote their releases as well.

### CODE REVIEW

File: layouts/shortcodes/redhat/latest-version.html [hunk 1]
```diff
@@ -1 +1 @@
-1.6.0
\ No newline at end of file
+1.9
```
Comment (thesuperzapper) [inline layouts/shortcodes/redhat/latest-version.html:1]: The new system (https://github.com/kubeflow/website/pull/3842) requires you to specify a specific version patch (it removes the patch version from the outer table but shows the full version inside the pop-up).

Please make this 1.9.0.

**Review by thesuperzapper — CHANGES_REQUESTED** [review]

**Comment by thesuperzapper** [comment]
Also, do you have a "version matrix" page which we can link to that lists which versions of Kubeflow components are in each Open Data Hub version?

For example, we use this one on deployKF: 
- https://www.deploykf.org/releases/tool-versions/#kubeflow-ecosystem

Alternatively, please at least link to your overall platform changelog (like we do for the other distributions).

**Review by tarilabs — APPROVED** [review]
/lgtm

and works with me to specify micro version too

**Review by terrytangyuan — APPROVED** [review]
/lgtm
/approve

**Pull request merged**